README.md

```diff
@@ -8,6 +8,11 @@ OpenBMC uses Yocto/Open-Embedded for a build system, which supports an
 out-of-tree build.  It is recommended that you create an empty directory
 somewhere to hold the build.  This directory will get big.
 
+On Fedora 23 the following packages are required to build the default target:
+
+    dnf install -y git patch diffstat texinfo chrpath SDL-devel bitbake
```

Comment:
I'm not sure that we will need bitbake from Fedora, as we use the built-in bitbake at https://github.com/openbmc/openbmc/blob/master/yocto-poky/bitbake/bin/bitbake

Have you tried to build on Fedora without bitbake installed?

Also, this commit will probably conflict with #121 so we should perhaps look to merge one first and then fix up this one.